checksum.go

```diff
@@ -5,13 +5,22 @@ package flagz
 
 import (
 	"hash/fnv"
+	"sync"
 
 	"github.com/spf13/pflag"
 )
 
+// @todo Temporary fix for race condition happening in the test:
+// spf13/pflag.FlagSet.VisitAll seems to be prone to a race condition
+// This fixes that but I'm not sure how much slower does it make the codebase
+var visitAllMutex = &sync.Mutex{}
+
 // ChecksumFlagSet will generate a FNV of the *set* values in a FlagSet.
 func ChecksumFlagSet(flagSet *pflag.FlagSet, flagFilter func(flag *pflag.Flag) bool) []byte {
 	h := fnv.New32a()
+
+	visitAllMutex.Lock()
```

Comment:
Admitteldy this feels heavy handed and not elegant considering the package mostly uses atomic instead of locking, but the reads and writes happening in `VisitAll` triggered the race detector.